README.md

```
@@ -10,8 +10,9 @@ By completing this assessment, you will:
 1. Understand the application's functionality by testing it locally.
 2. Learn to create a `Dockerfile` to containerize the application.
 3. Run the application inside a Docker container.
-4. Use Docker Compose to manage a multi-container setup.
+4. Use Docker Compose to manage a multi-container setup. 
 
+this for testing 
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)